### DISCUSSION TITLE
Could I use both v1 and v2 version?

### DISCUSSION THREAD

**Discussion by tylerrrkd** [heroui-inc/heroui#3093] [Help]
Developing a new features in an already exist project with `@nextui-org/react@1.0.0-beta.12`, and it was widely used in this project(hardly to upgrade globally). Furthermore, I need to use the new component `DatePicker` in v2. Is there any best practice to use both v1 and v2?

In addition, I have try to install single component package and its global dependencies like the following:
```
pnpm add @nextui-org/date-picker
pnpm add @nextui-org/system
pnpm add @nextui-org/theme
pnpm add framer-motion
pnpm add tailwindcss -D
```

Also followed the config https://nextui.org/docs/guide/upgrade-to-v2.

And the style looks weird.

Any suggestions? Thanks a lot.

**Comment by wingkwong** [comment]
We don't recommend to use both of them since there was major changes done in v2 and the dependencies are probably not compatible.

**Reply by tylerrrkd** [reply]
But, the problem is the v1 version was widely used in that old project and if I upgrade to v2, it will cost lot of time(such as `Grid`, `Loading` were replaced by in `v2`). Is there any suggestions if I just wanna use the `DatePicker` component in `v2`? Thanks for your replying, my friend.

**Comment by alphaxek** [comment]
Hi @tylerrrkd, Like wingkwong mentioned its just gonna create conflicts and issues.

but if you still wanna try test this `npm install @nextui-org/date-picker` `import {DatePicker} from "@nextui-org/date-picker";`
but again its not the correct way, would suggest to migrate to V2

Link:
https://nextui.org/docs/components/date-picker
